### ISSUE TITLE
Heat flux from obstruction

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#543]
```
Please complete the following lines...

Application Version: 5.2.4
SVN Revision Number:2651
Compile Date: 11 nov 2008
Operating System: Windows XP

Describe details of the issue below:
Hi!
I'm trying to run a very simple case: I simple want that the ambient 
temperature is set to 5 celsius and nothing more,all is fixed, so nothing 
should happen but what I see is that from the obstructions there is a neat 
heat flux that heats up the surrounding environment.
I'm puzzled...
It is something like the obstructions are some decimal of degrees hotter 
than the environment, but to make worse their wall temperature increase 
with time, but there isn't any kind of heat source.
Here is my code:

&HEAD CHID='box1',TITLE='box' /
&MESH IJK=40,32,20 XB=-0.200,0.8,-0.4,0.4,0.00,0.5 / Mesh 1




&TIME TWFIN=300.0 WALL_INCREMENT=1/
&MISC STRATIFICATION=.FALSE  TMPA=5  RADIATION=.FALSE./

&MATL ID = 'LDPE'
CONDUCTIVITY = 0.35
SPECIFIC_HEAT = 2.25
DENSITY = 920. / From Engineering materials 2 Ashby, Jones

&SURF ID = 'WALL'
MATL_ID ='LDPE'
RGB=138,43,226
BACKING ='EXPOSED'
THICKNESS = 0.025 
TMP_INNER=5/

&SURF ID = 'WIND'
PROFILE='ATMOSPHERIC'
VEL=0
PLE=0.10 
Z0=10 /  From meteo data (not yet) and from EPA for rural areas


&SURF ID='SLIP',SLIP_FACTOR=1.0 / Free-slip boundary




&OBST XB= 0.0,0.5,-0.35,0.35,0.0,0.025, SURF_ID='WALL'/
&OBST XB= 0.0,0.5,-0.35,0.35,0.4,0.425, SURF_ID='WALL'/
&OBST XB= 0.0,0.025,-0.35,0.35,0.0,0.425, SURF_ID='WALL'/
&OBST XB= 0.475,0.50,-0.35,0.35,0.0,0.425, SURF_ID='WALL'/
&OBST XB= 0.0,0.5,-0.35,-0.325,0.0,0.425, SURF_ID='WALL' COLOR='INVISIBLE'/
&OBST XB= 0.0,0.5,0.325,0.35,0.0,0.425, SURF_ID='WALL' COLOR='INVISIBLE'/

&VENT XB=-3,-3,-0.4,0.4,0,1.5 SURF_ID='WIND' /
&VENT XB=0.8,0.8,-0.4,0.4,0,1.5 SURF_ID='OPEN' /


&VENT MB='YMIN' ,SURF_ID='OPEN' /
&VENT MB='YMAX' ,SURF_ID='OPEN' /

&VENT XB=-3,0.8,-0.4,0.4,0,0 SURF_ID='INERT' / 
&VENT XB=-3,0.8,-0.4, 0.4, 1.5,1.5 SURF_ID='SLIP' /

&DEVC XB=0.05,0.45,-0.3,0.3,0,0.4   QUANTITY='TEMPERATURE' 
ID='temperature' STATISTICS='VOLUME MEAN' /

&SLCF PBY=0.0, QUANTITY='TEMPERATURE'   /

 &BNDF  QUANTITY='WALL_TEMPERATURE'/
&SLCF PBY=0.0, QUANTITY='VELOCITY' /


&TAIL /
```

Original issue reported on code.google.com by `giovanni.cocchi` on 2008-12-01 23:43:54

**Comment by gforney** [comment]
```
This was a bug in wall.f90. (Case RADIATION = .FALSE. lead to wrong heat flux terms
in Crank-Nicholson BC's). I have now fixed the bug and commited the updated wall.f90.


Can you compile your own code? Meanwhile, leave the radiation = .TRUE.. You can save
time in radiation solution by decreasing the number of radiation angles.

When you have a chance, please verify.
```

Original issue reported on code.google.com by `shostikk` on 2008-12-02 10:30:56

**Comment by gforney** [comment]
```
Ok, thank you! I will verify in the  next days and let you know. Actually I'm not 
able to compile the code, so I will choose the strategy of decreasing the number of

radiation angles.
May I suppose that the next manteinance release will have this bug fixed?
Best Regards
Giovanni Cocchi
```

Original issue reported on code.google.com by `giovanni.cocchi` on 2008-12-02 12:45:22

**Comment by gforney** [comment]
```
Yes, it will be. (since the fix is already in the source code repository)
```

Original issue reported on code.google.com by `shostikk` on 2008-12-02 12:56:24

**Comment by gforney** [comment]
```
All right, I'm running the case with radiation turned on with only 4 radiation 
angles and it's all right, both physically and numerically.
Thank you for your kind support.
Best regards
Giovanni Cocchi
```

Original issue reported on code.google.com by `giovanni.cocchi` on 2008-12-02 18:15:50

**Comment by gforney** [comment]
```
Giovanni, could you download the latest fds and try the case that originally didn't
work. Please confirm that the problem has been fixed.
```

Original issue reported on code.google.com by `shostikk` on 2008-12-16 13:24:07

**Comment by gforney** [comment]
```
I've run the case with the latest maintanence release and the problem is fixed.
Thank for your support.
Giovanni
```

Original issue reported on code.google.com by `giovanni.cocchi` on 2008-12-16 19:49:35

**Comment by gforney** [comment]
```
Thank you for letting us know.
```

Original issue reported on code.google.com by `mcgratta` on 2008-12-16 20:07:26

**Comment by gforney** [comment]
```
Sorry, my answer was premature...I've run another 2D no-fire case,but with radiation

turned on, and I got truly unphysical temperature field. Something like 60 degrees

in a room with inert wall and a little leakage of cold gas. 
I'm trying with the older release and temperature field is okay.
Giovanni

here is the code:
&HEAD CHID='propane_canalina_cr',TITLE='propane dispersion from continuos release'
/
&MESH IJK=125,1,112 XB=0.0,5,-0.001,0.001,-0.5,4, /
&TIME TWFIN=300.0 /
&RADI NUMBER_RADIATION_ANGLES=4
&MISC  BAROCLINIC=.TRUE./
&SPEC ID='PROPANE' /
&SURF ID='ADIAB' ADIABATIC=.TRUE COLOR='GRAY'/
&SURF ID='TUBO_ROTTO' COLOR='YELLOW'  MASS_FLUX(1)=0.1 TMP_FRONT=0 /
&OBST XB=0.25,5,-0.001,0.001,-0.5,0 SURF_ID='INERT'/
&OBST XB=0,0.04,-0.001,0.001,-0.5,-0.45 SURF_ID='TUBO_ROTTO'/ 

&VENT MB='XMIN' SURF_ID='MIRROR'/
&VENT XB=5,5,-0.001,0.001,3.5,4 SURF_ID='OPEN' /  Finestre aperte ai lati del 
capannone
&VENT MB='ZMAX' SURF_ID='ADIAB'/
&VENT MB='ZMIN' SURF_ID='ADIAB'/

&SLCF PBY=0.000,QUANTITY='TEMPERATURE' /
&SLCF PBY=0.000,QUANTITY='DENSITY' /
&SLCF PBY=0.000,QUANTITY='VELOCITY'  VECTOR=.TRUE /
&SLCF PBY=0.000,QUANTITY='PRESSURE' /
&SLCF PBY=0.000,QUANTITY='VORTICITY Y' /
&SLCF PBY=0.000,QUANTITY='PROPANE' /
&TAIL /
```

Original issue reported on code.google.com by `giovanni.cocchi` on 2008-12-16 23:08:40

**Comment by gforney** [comment]
```
Simo --  it appears that we introduced a bug in 5.2.5 with a forced flow boundary 
condition. I'll fix it.

Giovanni -- I suggest that you continue using 5.2.4
```

Original issue reported on code.google.com by `mcgratta` on 2008-12-17 13:43:24

**Comment by gforney** [comment]
```
Simo -- this bug has already been fixed by Jason, but it will not appear until the 
next release, which will be 5.3.0

Giovanni -- if you can compile the code, you can use the new version. Otherwise, it

will be a few weeks before the next release.
```

Original issue reported on code.google.com by `mcgratta` on 2008-12-17 14:48:32

**Comment by gforney** [comment]
```
I can't compile the code but for now I'm fine with the 5.2.4 release.
Thank You
Giovanni
```

Original issue reported on code.google.com by `giovanni.cocchi` on 2008-12-17 18:42:48

**Comment by gforney** [comment]
```
Giovanni -- can you confirm that your problem has been fixed in 5.3.0. Thanks.
```

Original issue reported on code.google.com by `mcgratta` on 2009-02-10 18:29:31

**Comment by gforney** [comment]
```
Assumed Verified
```

Original issue reported on code.google.com by `mcgratta` on 2009-02-18 22:30:51
